### ISSUE TITLE
Ability to build without JCenter

### ISSUE THREAD

**Issue by szpak** [gradle-nexus/publish-plugin#56]
JCenter is going to be shut down "soon" (although 1st May has been "clarified" to 1st Feb 2022 :-D ). When I tried to remove `jcenter()` the build on CI server started to [fail](https://github.com/gradle-nexus/publish-plugin/runs/1875382589?check_suite_focus=true#step:4:56):

```
> Task :dokkaJavadoc FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':dokkaJavadoc'.
> Could not resolve all files for configuration ':dokkaJavadocPlugin'.
   > Could not find org.jetbrains.dokka:javadoc-plugin:1.4.20.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/jetbrains/dokka/javadoc-plugin/1.4.20/javadoc-plugin-1.4.20.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':dokkaJavadoc'.
	at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:38)
```

In Maven Central, there is a similar version available, so probably it is possible to override transitive dependencies, however, it would be best to fix it upstream - https://github.com/Kotlin/dokka/issues/1725

**Comment by szpak** [comment]
With an exception for dokka itself, there are more dokka-related transitive dependencies missing:
```
* What went wrong:
Execution failed for task ':dokkaJavadoc'.
> Could not resolve all files for configuration ':dokkaJavadocPlugin'.
   > Could not find com.soywiz.korlibs.korte:korte-jvm:1.10.3.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/soywiz/korlibs/korte/korte-jvm/1.10.3/korte-jvm-1.10.3.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project : > org.jetbrains.dokka:javadoc-plugin:1.4.20
   > Could not find org.jetbrains.kotlinx:kotlinx-html-jvm:0.7.2.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/jetbrains/kotlinx/kotlinx-html-jvm/0.7.2/kotlinx-html-jvm-0.7.2.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project : > org.jetbrains.dokka:javadoc-plugin:1.4.20
         project : > org.jetbrains.dokka:javadoc-plugin:1.4.20 > org.jetbrains.dokka:dokka-base:1.4.20
```

And later:
```
* What went wrong:
Execution failed for task ':dokkaJavadoc'.
> Could not resolve all files for configuration ':dokkaJavadocRuntime'.
   > Could not find org.jetbrains:markdown:0.1.45.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/org/jetbrains/markdown/0.1.45/markdown-0.1.45.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project : > org.jetbrains.dokka:dokka-core:1.4.20
```

**Comment by nhajratw** [comment]
I think this has been resolved with the fixing of #57

**Comment by szpak** [comment]
Good catch, thanks.
